__tests__/pages/index.test.tsx

```diff
@@ -11,11 +11,11 @@ describe("HomePage", () => {
   it("renders the last updated time correctly", () => {
     render(<HomePage latestNews={[latestNewsItemBuilder()]} />);
 
-    expect(
-      screen.getByText(
-        /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
-      ),
-    ).toBeVisible();
+    const LastUpdatedAlert = screen.queryByText(
+      /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
+    );
+
+    expect(LastUpdatedAlert).not.toBeInTheDocument();
   });
 });
```

Comment:
I think we can remove this test, wdyt @zainfathoni @antoooks?

Reply:
Good point, actually this is a question from me as well. If we remove a component altogether from existing page, should we test the if the component not exist or delete the test altogether?

Reply:
Removed, I think we can resolve this conversation